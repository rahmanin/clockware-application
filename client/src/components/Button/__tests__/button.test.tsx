import {Button} from "../index"
import React from 'react';
import {cleanup, render} from '@testing-library/react';

afterEach(cleanup);

test('Render Button', () => {
  const component = render(
    <Button title={"Aaa"} disabled={false} color={"black"}/>,
  );
  expect(component.baseElement).toMatchSnapshot();
  
});