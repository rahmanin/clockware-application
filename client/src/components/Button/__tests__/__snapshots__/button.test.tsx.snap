// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Button 1`] = `
<body>
  <div>
    <button
      class="button button--black"
    >
      Aaa
    </button>
  </div>
</body>
`;
